limit-pull-requests/action.yml

```diff
@@ -0,0 +1,82 @@
+name: Limit pull requests
+description: Limit the number of pull requests to the repository created by a user
+author: ZhongRuoyu
+branding:
+  icon: alert-triangle
+  color: yellow
+
+inputs:
+  token:
+    description: GitHub token
+    required: false
+    default: ${{ github.token }}
+  limit:
+    description: The maximum number of pull requests allowed
+    required: true
+    default: "10"
+  except-users:
+    description: The users exempted from the limit, one per line
+    required: false
+  # https://docs.github.com/en/graphql/reference/enums#commentauthorassociation
+  except-author-associations:
+    description: The author associations exempted from the limit, one per line
```

Comment:
Do you have an example of how this might work/be used?

Reply:
```yaml
- uses: Homebrew/actions/limit-pull-requests@master
  with:
    limit: 10
    except-author-associations: |
      COLLABORATOR
      MEMBER
    comment: >
      Blah blah
```

Within Homebrew, probably not if we're going to apply this limit to everyone (including maintainers) except @BrewTestBot. However, any potential non-Homebrew users of this action can exempt collaborators, maintainers, etc. from the limit.

I can remove this if this is deemed unneeded.

Reply:
Gotcha, thanks. Might want to add this to docs?

Reply:
Added in 6c9125d